sentry_sdk/tracing_utils.py

```diff
@@ -152,7 +153,18 @@ def record_sql_queries(
 
 
 def maybe_create_breadcrumbs_from_span(hub, span):
```

Comment:
should we maybe change the `hub` parameter's name to `scope`? I suppose it would be a breaking change if someone was using keyword arguments